modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java

```diff
@@ -1861,6 +1974,36 @@ public int execute(List<String> rawArgs) {
                     new SecurityCredentialsBasicProvider(new SecurityCredentials(args.user(), args.password())));
             }
 
+            if (args.isSslEnable()){
```

Comment:
whitespace desired

Reply:
Can you move all the following into a separate method? It becomes too long otherwise.